src/app/components/profil/profil.component.scss

```diff
@@ -0,0 +1,102 @@
+@use '../../../import' as *;
+
+section{
+    width: 90%;
+    margin: 5% auto;
+    background-color: $main-background-color;
+    display: flex;
+    justify-content: space-around;
+    padding-top: 2%;
+    padding-bottom: 2%;
+
+    .user-img{
+        width: 40%;
+        img{
+            width: 100%;
+        }
+    }
+    .user-description{
+        width: 50%;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-around;
+        align-items: center;
+        h3{
+            color: $dark-text-color;
+            font-size: 1.5em;
+        }
+        p{
+            color: $dark-text-color;
+            font-size: 1em;
+        }
+
+        
+    button{
+        background-color: $button-color;
+        color: $light-text-color;
+        padding: 1%;
+        border: none;
+        border-radius: 10px;
+        cursor: pointer;
+        width: 50%;
+    }
+
+    }
+}
+
+@media screen and (min-width: 721px) {
+    section{
+        width: 50%;
+        margin: 5% 10% 5% 40%;
+        background-color: $main-background-color;
+        display: flex;
+        justify-content: space-around;
+        padding-top: 2%;
+        padding-bottom: 2%;
+    
+        .user-img{
+            width: 40%;
+            img{
+                width: 100%;
+            }
+        }
+        .user-description{
+            width: 50%;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-around;
+            align-items: center;
+            h3{
+                color: $dark-text-color;
+                font-size: 1.5em;
+            }
+            p{
+                color: $dark-text-color;
+                font-size: 1em;
+            }
+    
+            
+        button{
+            background-color: $button-color;
+            color: $light-text-color;
+            padding: 1%;
+            border: none;
+            border-radius: 4%;
+            cursor: pointer;
+            width: 50%;
+        }
```

Comment:
The indentation of button is different from the others, make it match with p and h3.